enterprise_subsidy/apps/api/v1/serializers.py

```diff
@@ -74,6 +74,7 @@ class TransactionSerializer(serializers.ModelSerializer):
         help_text="The unit in which this transaction's quantity is denominated."
     )
     reversal = ReversalSerializer(read_only=True)
+    metadata = serializers.SerializerMethodField()
```

Comment:
can we add a comment here pointing at https://romansorin.com/blog/using-djangos-jsonfield-you-probably-dont-need-it-heres-why ?